website/print/docs/racer/license-back/.document.inc

```diff
@@ -0,0 +1,69 @@
+<?php
+
+require_once('inc/partitions.inc');
+require_once('inc/path-info.inc');
+require_once('inc/photo-config.inc');
+
+class DriversLicenseBackDocument extends PrintableRacerDocument {
+  function name() { return "Driver's License Back"; }
+
+  function get_available_options() {
+    return array(
+      'use_photo' => array('type' => 'bool',
+                           'desc' => 'Include racer\'s photo',
+                           'default' => true),
+      'use_barcode' => array('type' => 'bool',
```

Comment:
These two options (use_barcode and date_of_issue) should be removed, please, as they aren't meaningful for this document.